modules/extensions/src/collision-filter/collision-filter-effect.ts

```diff
@@ -244,18 +243,23 @@ export default class CollisionFilterEffect implements Effect {
       }
     });
 
-    const depthBuffer = new WEBGLRenderbuffer(
-      // @ts-expect-error TODO v9 needs to be WebGLDevice
-      device,
-      {format: 'depth16unorm', width, height}
-    );
+    // @ts-ignore
+    const depthStencilAttachment = device.createTexture({
+      format: 'depth16unorm',
+      width,
+      height,
+      mipmaps: false,
+
+      // TODO fix getWebGLTextureParameters() in luma to avoid passing deprecated parameters
+      dataFormat: 6402, // gl.DEPTH_COMPONENT
```

Comment:
My understanding is that these unsized formats should be avoided in WebGL2, also see the stack overflow link in the slack channel. You may want `depth24unorm`  (`GL.DEPTH_COMPONENT24`?) for better precision